docs/source/x_ext.rst

```diff
@@ -1052,6 +1060,8 @@ allowed to provide a result for an instruction once the core has indicated (via
 have exactly one result transaction (even if no data needs to be written back to the |processor|'s register file). No result transaction shall be performed for instructions which have not been accepted for offload or
 for instructions that have been killed.
 
+If ``result_valid`` is 0, all signals within ``result`` must also be 0.
```

Comment:
Same comment as above.
```suggestion
If ``result_valid`` is 0, all signals within ``result`` must be ignored.
```
Forcing them to 0 is too restrictive and may prevent power optimizations. Simply ignoring them opens the door to more design choices.
In the multiple coprocessor scenario, it is up to the signal combination logic to deal with this, based on the choice that was made by each coprocessor provider.

Reply:
point taken. i would like to get this discussed in a wider audience.